realm/realm-library/src/main/cpp/io_realm_internal_OsObject.cpp

```diff
@@ -26,11 +26,15 @@
 #include "jni_util/java_global_weak_ref.hpp"
 #include "jni_util/java_method.hpp"
 #include "jni_util/java_class.hpp"
+#include "jni_util/java_exception_thrower.hpp"
 
 using namespace realm;
 using namespace realm::jni_util;
 using namespace realm::_impl;
 
+static const char* PK_CONSTRAINT_EXCEPTION_CLASS = "io/realm/exceptions/RealmPrimaryKeyConstraintException";
```

Comment:
I still think it would be a great idea to put them all in one place.  Suppose that I am about to add a PK_CONSTRAINT_EXCEPTION to the code.  Why would I go looking in io_real_internal_OsObject.cpp, to find it?